core/src/main/scala/frameless/Ordered.scala

```diff
@@ -0,0 +1,17 @@
+package frameless
+
+/** Types that can be ordered/compared by Spark. */
+trait Ordered[A]
```

Comment:
Hmm the name conflicts with `scala.math.Ordered`... What about something like `SparkOrdered` or `Ordered4Spark` instead? If you change it, you might also want to update [Averagable](https://github.com/adelbertc/frameless/blob/master/dataset/src/main/scala/frameless/functions/Averagable.scala) and [Summable](https://github.com/adelbertc/frameless/blob/master/dataset/src/main/scala/frameless/functions/Summable.scala).